sale_invoice_plan/models/sale.py

```diff
@@ -13,7 +13,7 @@ class SaleOrder(models.Model):
     invoice_plan_ids = fields.One2many(
         comodel_name="sale.invoice.plan",
         inverse_name="sale_id",
-        string="Inovice Plan",
+        string="Invoice Plan",
```

Comment:
Just a little comment.
no need to add because odoo will auto use string `Invoice Plan`